From 4474653efed971889a58193a67b78f1bf2cab312 Mon Sep 17 00:00:00 2001
From: Sergio Costas <raster@rastersoft.com>
Date: Wed, 28 Nov 2018 10:01:23 +0100
Subject: [PATCH 10/55] FileItem.js: fix access to destroyed trash icon

Due to a typo, the trash folder monitor was never disconnected when
the desktop was recreated, so an "invalid access" error was produced
every time the trash folder was modified.

This patch fixes the typo.
---
 fileItem.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fileItem.js b/fileItem.js
index f927efa..abf6ded 100644
--- a/fileItem.js
+++ b/fileItem.js
@@ -177,7 +177,7 @@ var FileItem = class {
             this._loadThumbnailDataCancellable.cancel();
 
         /* Trash */
-        if (this._monitorTrashDirId)
+        if (this._monitorTrashDir)
             this._monitorTrashDir.disconnect(this._monitorTrashId);
         if (this._queryTrashInfoCancellable)
             this._queryTrashInfoCancellable.cancel();
-- 
2.20.0

